### ISSUE TITLE
Polish Multiclosure and Closure test module

### ISSUE THREAD

**Issue by comane** [NNPDF/nnpdf#2159]
Multiclosure test module need to be polished as several functions are either obsolete or unused

**Comment by RoyStegeman** [comment]
What is the status of the existing PRs? If they're in good shape we don't have to wait until the paper has been published before merging. That may be more convenient than having to keep track of many different branches

**Comment by comane** [comment]
Yes I agree, the PR #1982 that is used for the analysis is almost ready to be merged. I will take care of it before starting anything else.
